src/agent/Cargo.toml

```diff
@@ -19,3 +19,6 @@ members = [
 
 [profile.release]
 lto = "thin"
+
+[patch.crates-io]
+appinsights = { git = "https://github.com/Porges/appinsights-rs", branch="audit-fixes" }
```

Comment:
You should link your PR so we know that when it's released we can go back to the 'official' crate

Reply:
It's in the commit message but I'll add it here too.